### PR TITLE
Upgrade/800/server hardware

### PR DESCRIPTION
**Pull request by sijeesh**
### Description
Upgraded Server Hardware resource to support OneView API version 800

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/server_hardware.py [hunk 1]
```diff
@@ -44,43 +43,22 @@
...
 for serv in server_hardware_all:
     print('  %s' % serv['name'])
 
-# Create a rack-mount server
-server = oneview_client.server_hardware.add(options)
-print("Added rack mount server '%s'.\n  uri = '%s'" %
-      (server['name'], server['uri']))
+server = server_hardwares.get_by_name(server_name)
+if not server:
+    # Create a rack-mount server
```
Comment (soodpr) [inline examples/server_hardware.py]: its better to update the comment that it works only for Synergy

File: examples/server_hardware.py [hunk 2]
```diff
@@ -44,43 +43,22 @@
...
+    server = server_hardwares.add(options)
+    print("Added rack mount server '%s'.\n  uri = '%s'" % (server['name'], server['uri']))
 
 # Create Multiple rack-mount servers
```
Comment (soodpr) [inline examples/server_hardware.py]: same as above

File: hpOneView/resources/servers/server_hardware.py [hunk 3]
```diff
@@ -43,118 +44,8 @@ class ServerHardware(object):
...
-                    5-minute sample value data during the day. Samples at this resolution are retained up to three
-                    years.
-
-        Returns:
-            dict
-        """
-
-        return self._client.get_utilization(id_or_uri, fields=fields, filter=filter, refresh=refresh, view=view)
-
-    def get_all(self, start=0, count=-1, filter='', sort=''):
```
Comment (soodpr) [inline hpOneView/resources/servers/server_hardware.py]: we may have to overwrite this method as done in logical enclosures to scopeUris as query param.

Comment (sijeesh) [reply hpOneView/resources/servers/server_hardware.py]: There is no scopeUris param for server hardware.

File: examples/server_hardware.py (hunk 1 above)
Comment (sijeesh) [reply examples/server_hardware.py]: Updated the comment

File: examples/server_hardware.py (hunk 2 above)
Comment (sijeesh) [reply examples/server_hardware.py]: Updated the comment

**Comment by sijeesh** [comment]
> server_mpHostsAndRanges is missing in the config file



> server_mpHostsAndRanges is missing in the config file
Update

**Comment by sijeesh** [comment]
> server_mpHostsAndRanges is missing in the config file

Added server_mpHostsAndRanges  to config

File: hpOneView/resources/servers/server_hardware.py (hunk 3 above)
Comment (soodpr) [reply hpOneView/resources/servers/server_hardware.py]: My Bad. Looks like I have mistakenly checked the POST endpoint where we have ScopeUris field.

**Comment by sijeesh** [comment]
> > iloSsoUrl
> 
> This parameter is there from API version 500, not sure why it's not added, anyways let me have a look.

Updated methods with Query parameter.

**Pull request merged**